src/installer/pkg/sfx/Microsoft.NETCore.App/PackageOverrides.txt

```diff
@@ -132,31 +154,37 @@ System.Runtime.Numerics|4.3.0
 System.Runtime.Serialization.Formatters|4.3.0
 System.Runtime.Serialization.Json|4.3.0
 System.Runtime.Serialization.Primitives|4.3.0
-System.Security.AccessControl|4.4.0
+System.Runtime.Serialization.Xml|4.3.0
+System.Security.AccessControl|6.0.1
 System.Security.Claims|4.3.0
-System.Security.Cryptography.Algorithms|4.3.0
-System.Security.Cryptography.Cng|4.4.0
+System.Security.Cryptography.Algorithms|4.3.1
+System.Security.Cryptography.Cng|5.0.0
 System.Security.Cryptography.Csp|4.3.0
 System.Security.Cryptography.Encoding|4.3.0
-System.Security.Cryptography.OpenSsl|4.4.0
+System.Security.Cryptography.OpenSsl|5.0.0
 System.Security.Cryptography.Primitives|4.3.0
-System.Security.Cryptography.X509Certificates|4.3.0
-System.Security.Cryptography.Xml|4.4.0
+System.Security.Cryptography.X509Certificates|4.3.2
 System.Security.Principal|4.3.0
-System.Security.Principal.Windows|4.4.0
+System.Security.Principal.Windows|5.0.0
+System.Security.SecureString|4.3.0
 System.Text.Encoding|4.3.0
+System.Text.Encoding.CodePages|9.0.0
 System.Text.Encoding.Extensions|4.3.0
-System.Text.RegularExpressions|4.3.0
+System.Text.Encodings.Web|9.0.0
+System.Text.Json|9.0.0
```

Comment:
These `9.0.0` ones are interesting. Is there a reason they weren't included to begin with?

Reply:
System.Text.Json is a new package but System.Text.Encodings.Web existed since 2015 and has been inbox since at least netcoreapp2.0. If I'm not mistaken it was you who created the initial list?

Reply:
This list hasn't changed since you originally added it.  I think we were letting platform manifest do the work then.

With PackageOverrides we can short circuit that work.  It can also help if we eventually feed this data into NuGet for supplied by platform (short circuiting even earlier in restore).

Reply:
> If I'm not mistaken it was you who created the initial list?

I believe it was initially added in https://github.com/dotnet/core-setup/pull/7369/.

Reply:
That was just copied from your list: https://github.com/dotnet/core-setup/pull/7369/files#diff-d43a6b44d056f972850070a59b2db1c58d30dd708de5b8a27f492a4b97d71210R4